tests/client/test_search_results.py

```diff
@@ -1,54 +0,0 @@
-import pytest
-from lxml import etree
-
-from caselawclient.models.search_results import SearchResults
-
-
-class TestSearchResults:
```

Comment:
These get added back in https://github.com/nationalarchives/ds-caselaw-custom-api-client/pull/223/files#diff-963e91dbe51e70ef86a30719fdb422d993fdb30e1a553bb10e444222c1f072c7